docs/orchestrating-elastic-stack-applications/managing-compute-resources.asciidoc

```diff
@@ -92,6 +92,63 @@ spec:
 
 For the container name, you can use `apm-server` or `kibana` as appropriate.
 
+[float]
+[id="{p}-compute-resources-beats-agent"]
+=== Set compute resources for Beats and Elastic Agent
+
+For Beats or Elastic Agent objects, the `podTemplate` can be configured as follows, depending on the chosen deployment model.
+
+When deploying as a Kubernetes Deployment:
```

Comment:
```suggestion
To deploy as a Kubernetes Deployment:
```

Reply:
I am trying to say: "if you deploy as a Kubernetes Deployment you have to define the resource spec like this", so "Deployment" is already a given.